css-brochure-form/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Learn More</h1>
+    <form action="" method="post">
+      <div>
+        <label for="email"
+          >Company Email<input
+            id="email"
+            type="email"
+            name="email"
+            placeholder="sally@company.co"
+        /></label>
+      </div>
+      <div>
+        <label for="name"
+          >Name<input id="name" name="email" placeholder="Sally Smith"
+        /></label>
```

Comment:
Minor thing here I think you meant to have `name="name"` since you have your email input above.

Reply:
Thanks! Just made the fix!